test/integration/builtin/filters/date.ts

```diff
@@ -12,7 +12,19 @@ describe('filters/date', function () {
     return test('{{ "today" | date: "%Y"}}', (new Date()).getFullYear().toString())
   })
   it('should parse as Date when given UTC string', function () {
-    return test('{{ "1991-02-22T00:00:00" | date: "%Y"}}', '1991')
+    return test('{{ "1991-02-22T00:00:00" | date: "%Y-%m-%dT%H:%M:%S"}}', '1991-02-22T00:00:00')
+  })
+  it('should not change the timezone between input and output', function () {
+    return test('{{ "1990-12-31T23:00:00Z" | date: "%Y-%m-%dT%H:%M:%S"}}', '1990-12-31T23:00:00')
+  })
+  it('should apply numeric offset', function () {
+    return test('{{ "1990-12-31T23:00:00+00:00" | date: "%Y-%m-%dT%H:%M:%S"}}', '1990-12-31T23:00:00')
+  })
+  it('should apply numeric offset', function () {
+    return test('{{ "1990-12-31T23:00:00-01:00" | date: "%Y-%m-%dT%H:%M:%S"}}', '1990-12-31T23:00:00')
```

Comment:
TODO mock the local timezone offset for these

Reply:
This might be quite difficult to mock actually :/ Just overriding `Date.prototype.getTimezoneOffset` doesn't affect how the constructor processes date string inputs

Reply:
If I understand correctly, the idea is: making `{{ "<datestr>" | date: "%Y-%m-%dT%H:%M:%S" }}` always equal to `<datestr>`. It's OK to not mock a different timezone in test. The travis machine is already in a different timezone, most likely.

Reply:
Yeah, for most of the non-UK developers the test should be fine 😁